Arrangement_on_surface_2/include/CGAL/Arr_dcel_base.h

```diff
@@ -930,13 +961,18 @@ class Arr_dcel_base {
   Inner_ccb_allocator in_ccb_alloc;         // An allocator for inner CCBs.
   Iso_vert_allocator  iso_vert_alloc;       // Allocator for isolated vertices.
 
+#ifndef CGAL_ARRANGEMENT_DISABLE_UNION_SET_CCB
+  Inner_ccb_uf        inner_ccb_uf;         // efficient CCB merging
```

Comment:
you probably want to clear the container when you're done with the construction of the arrangement.

Reply:
Indeed, I'll change that.